### PR TITLE
Feature/homepage

### PR DESCRIPTION
**Pull request by Prakort**

### CODE REVIEW

File: app/javascript/style/application.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import "bootstrap/scss/bootstrap";
```
Comment (amanpatel123) [inline app/javascript/style/application.scss:1]: Nicee!!

**Comment by amanpatel123** [comment]
Wow, this is amazing!!!!
image
@Prakort this is neat work. I am impressed

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   root to: 'root#index'
 
-  # get "/repository/*path", to: 'root#index'
+  get "/*path", to: 'root#index'
```
Comment (amanpatel123) [inline config/routes.rb:4]: @Prakort just added this

**Review by amanpatel123 — APPROVED** [review]

**Comment by HarshdipD** [comment]
Awesome @Prakort !!

**Pull request merged**